package-lock.json

```diff
@@ -1892,10 +1892,10 @@
       "integrity": "sha1-DQcLTQQ6W+ozovGkDi7bPZpMz3c=",
```

Comment:
Can we move the `package-lock.json` changes to a separate card?

Reply:
Alright, resolved

Reply:
Thanks 👍